content/millennium/r4/financial.md

```diff
@@ -10,6 +10,7 @@ layout: api
 <%= render '/millennium/r4/financial/support.*' %>
 
 * [Coverage](/millennium/r4/financial/support/coverage)
+* [InsurancePlan](/millennium/r4/financial/support/insurance-plan)
```

Comment:
Please move the documentation under /millennium/r4/financial/general

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/809/commits/f285387f72f531a780abd964a8c1b3335e9d5385#diff-0eeb022ddf17088bd7f896481e3b1ee471fe27e9abaf286a5a6b3e22bbfb97a9R21